src/itype.h

```diff
@@ -648,8 +649,9 @@ struct itype {
 
     bool rigid = true; // If non-rigid volume (and if worn encumbrance) increases proportional to contents
 
-    int melee_dam = 0; // Bonus for melee damage; may be a penalty
-    int melee_cut = 0; // Cutting damage in melee
+    /** Damage output in melee for zero or more damage types */
+    std::array<int, NUM_DT> melee;
```

Comment:
Is it zeroed properly somewhere?
I created at least 2 bugs by not explicitly zeroing arrays.